Provider.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,12 +1,13 @@
 {
+  "originHash" : "e89086004bb1eb0eadde723b71e9d5a8a0b8b07ce2d76f38307d64c2244fd5ba",
   "pins" : [
     {
       "identity" : "networking",
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/Lickability/Networking",
       "state" : {
         "branch" : "main",
-        "revision" : "bf1a3f965845154a1553355a1124964bcdc62243"
+        "revision" : "c7013a352d5a52ed433a34b7af50b3bc008d37cb"
```

Comment:
I'm thinking that this maybe should use a tagged release over a commit of main.

Reply:
ok